docs/conf.py

```diff
@@ -123,7 +123,7 @@
 # documentation.
 html_theme_options = {
     "announcement": "⚠ python-gitlab 3.0.0 has been released with several "
-    "<a href='changelog.html#breaking'>breaking changes</a>.",
+    "<a href='/changelog.html#breaking'>breaking changes</a>.",
 }
```

Comment:
Thanks for the catch @wadimklincov! It's served on RTD mainly so this should work.

But I'm thinking v3 has been out for a month and we follow semver so people should know by now, this banner was always meant to be temporary. Maybe we could remove it? So we could just revert https://github.com/python-gitlab/python-gitlab/commit/a349793307e3a975bb51f864b48e5e9825f70182 :)

Reply:
Sure @nejch, sounds good to me!

Reply:
@nejch Replaced it with the reverted commit now. Should I update the PR message? Not sure how it works on Github!

Reply:
Thanks for that @wadimklincov. I edited the title but the important part is just the commit message as we use it for changelogs, so I edited it a bit to make sure `python-semantic-release` can parse it correctly.

BTW: if you add your Siemens email to your github account it'll link your contributions there under your account, currently I think it shows as unverified :)

Reply:
Ah, thanks for updating it @nejch, totally missed that!

Interesting, I'll try that, thank you!